crates/moonbuild/src/entry.rs

```diff
@@ -519,6 +519,9 @@ pub fn run_test(
                     Err(e) => {
                         // when spawn process failed, this can still make the total test count to be correct
                         // but this is not a good way to handle it
+                        for _ in 0..test_args.get_test_cnt() {
```

Comment:
maybe we should just show the error message only once?

Reply:
> maybe we should just show the error message only once?

When much errors occur, your suggestion can reduce some of them. But when there are several(rare) errors, the user may be confused by the failed count and concrete error messages. I am OK with your suggestion if you think the unmatched count is acceptable.

Reply:
yeah I think showing once is ok